add new command (#5855)
dbt-labs/docs.getdbt.com: website/docs/docs/build/metricflow-commands.md

- [`list entities`](#list-entities) &mdash; Lists all unique entities.
- [`list saved queries`](#list-saved-queries) &mdash; Lists available saved queries. Use the `--show-exports` flag to display each export listed under a saved query.
- [`query`](#query) &mdash; Query metrics, saved queries, and dimensions you want to see in the command line interface. Refer to [query examples](#query-examples) to help you get started.

> - [`export`](#exports) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.

Author: ```suggestion
- [`export`](#export) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.
```

> - [`export`](#exports) &mdash;  Runs runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.

Reviewer: ```suggestion
- [`export`](#exports) &mdash;  Runs exports for a singular saved query for testing and generating exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query.
```

Author: nice catch, thank yoU!

mf query --metrics revenue --group-by metric_time__month # In dbt Core
```

> ### Exports

Reviewer: Think this should be singular like the command
```suggestion
### Export
```

> Run [exports for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use this command to test and generate exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.

Reviewer: Same as above
```suggestion
Run [export for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use this command to test and generate exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.
```

> Run [exports for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use this command to test and generate exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.

Reviewer: Similar to above, I think we should reference the command as the singular 
```suggestion
Run the [`export` command for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use it in your development environment to test and generate exports. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.
```

```bash
dbt sl export # In dbt Cloud

> mf export # In dbt Core

Reviewer 2: This isn't a thing in OSS, as exports are a cloud limited feature

Author: Oh yes, great shout @WilliamDee !

```

### Export-all

Run [exports for multiple saved queries](/docs/use-dbt-semantic-layer/exports#exports-for-multiple-saved-queries) at once. This command provides a convenient way to manage and execute exports for several queries simultaneously, saving time and effort. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info.

```bash
dbt sl export-all # In dbt Cloud

> mf export-all # In dbt Core

Reviewer 2: ditto on the not supported in OSS

mf query --metrics revenue --group-by metric_time__month # In dbt Core
```

### Export

Run [exports for a specific saved query](/docs/use-dbt-semantic-layer/exports#exports-for-single-saved-query). Use this command to test and generate exports in your development environment. You can also use the `--select` flag to specify particular exports from a saved query. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info. 

> Exports is available in dbt Cloud.

Author: ```suggestion
Export is available in dbt Cloud.
```

```bash
dbt sl export 
```

### Export-all

Run [exports for multiple saved queries](/docs/use-dbt-semantic-layer/exports#exports-for-multiple-saved-queries) at once. This command provides a convenient way to manage and execute exports for several queries simultaneously, saving time and effort. Refer to [exports in development](/docs/use-dbt-semantic-layer/exports#exports-in-development) for more info. 

> Exports is available in dbt Cloud.

Author: ```suggestion
Export is available in dbt Cloud.
```
